### ISSUE TITLE
Geometry - Mechanical Interference

### ISSUE THREAD

**Issue by tshumay** [TheRobotStudio/Dexhand#7]
Especially with some deviation in 3D printing, there is a mechanical interference with the top right tendon port and the servo mounting positions on the forearm.

<img width="1071" alt="Screenshot 2023-06-11 at 8 08 57 AM" src="https://github.com/TheRobotStudio/Dexhand/assets/2821763/5935163f-6ac1-4e68-ac08-aab9cb86d2fb">

**Comment by TheRobotStudio** [comment]
Yes - this is intended to be cleared with a 2mm drill prior to assembly!

**Comment by tshumay** [comment]
It's not the hole itself, so much as the tendon being impeded by the servo body. I cleared the holes, but it's obscured by the mount tail of the servo.
